python/cudf/dataframe/dataframe.py

```diff
@@ -1865,20 +1729,19 @@ def apply_chunks(self, func, incols, outcols, kwargs={}, chunks=None,
         ``range(cuda.threadIdx.x, in1.size, cuda.blockDim.x)``, the *kernel*
         function can be used with any *tpb* in a efficient manner.
 
-        .. code-block:: python
+        >>> from numba import cuda
 
```

Comment:
For a single example, when there are newlines between code, numpydoc renders them as separate code blocks.

This could be my stylistic preference, but removing blank lines so that numpydoc renders the full example in a single code-block would be clearer.

Reply:
Done.  In some cases where there were comments in between I elevated those comments to prose and kept the line break.  In the cases that you're highlighting though they've been merged into a single code block.